BPForbes_Flinstone_Tests.c

```
@@ -1019,6 +1079,34 @@ void test_interpreter_setdisk_allowed_inside_jail(void) {
     remove(diskfile);
 }
 
+void test_diskput_fat32_roundtrip(void) {
+    char prev[CWD_MAX];
+    strncpy(prev, current_disk_file, sizeof(prev));
+    prev[sizeof(prev) - 1] = '\0';
+    FILE *fp = fopen("bin_src.dat", "wb");
+    CU_ASSERT_PTR_NOT_NULL(fp);
+    unsigned char b[] = {0, 1, 2, 255, 10, 20};
```

Clang (14.0.6)

[warning] variable name 'b' is too short, expected at least 3 characters

(readability-identifier-length)